cmd/start.go

```diff
@@ -24,6 +24,7 @@ const (
 	bearerTokenFlagName    = "bearer-token"
 	corsFlagName           = "cors-origin"
 	syncProviderFlagName   = "sync-provider"
+	prettyLogFlagName      = "prettylogger"
```

Comment:
can we update this flag name to something a little clearer, such as `log-format`, with this value a developer would have to pass `prettyLogger=json`